packages/desktop-client/src/components/reports/graphs/net-worth-spreadsheet.tsx

```diff
@@ -152,5 +163,7 @@ function recalculate(data, start, end) {
     },
     netWorth: endNetWorth,
     totalChange: endNetWorth - startNetWorth,
+    lowestNetWorth: lowestNetWorth,
+    highestNetWorth: highestNetWorth,
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
    lowestNetWorth,
    highestNetWorth,
```